compiler/control/OMROptions.cpp

```diff
@@ -1869,6 +1870,16 @@ OMR::Options::Options(
          optimizationPlan->setOptLevelDowngraded(false);
          }
       }
+   // Following set of options are needed as a workaround to enforce vector API
+   // expansion of Vector API intrinsics for each platform.
+   // TODO: This enforce option is added temporary to test JIT exploitation of
+   // Vector API, clean up is needed once we have resolved all workarounds.
+   if (self()->getOption(TR_EnforceVectorAPIExpansion))
+      {
+      self()->setOption(TR_DisableOSRGuardMerging);
```

Comment:
Isn't this option only needed on X and Z?

Reply:
@BradleyWood  Yes as Z and X are the only ones where we merge OSR guards. By default ARM and Power returns false and actually does not check for the option at all. 
I have not put the platform guard here and kept the options that I found working on Z and probably x86 (Based on the internal issues documenting the test). 
Additional options needed for other platform can be added on top of this changes with appropriate platform guard.